Lineage 15.1

sassmann opened the pull request.

Commit:
lentis-common: fingerprint: Move device dependent modules to /vendor

Change-Id: I7a28b65a55cf808a01b6defdce1f7ff1f64bc565

sam3000 committed:
lentis-common: Overlays: Update for new LED capabilities

*O
) Use LIGHTS_BATTERY_LIGHT capability in lineage-sdk instead.

*) Update config_deviceLightCapabilities comments

Change-Id: Ia813cdd0596ece07ff38e1087d9cdf9e135f20be

haggertk committed:
lentis-common: fingerprint: Bump version to 2.1

Change-Id: Id355acaf78387917252c56476da0ff8d3828f5d0

haggertk committed:
lentis-common: fingerprint: Resolve compilation warnings

* fingerprint_enumerate had an incorrect signature
* Quit partially initializing fingerprint_msg_t instances

Change-Id: I09a05f3656c5bdc1e17fa76b5e68c56e0b7277de

sassmann committed:
lentis-common: user powersave governor for debugging

Change-Id: I1c478b772eaf4afb6d02642f7d15fe37a4eedd08

sassmann committed:
lentis-common: update services path to /system/vendor (--pedantic)

Change-Id: I3aaf0bd1f9990e42fb755557751c59a0af6d2ca3

sassmann committed:
lentis-common: remove fingerprintd service

use fingerprint 2.0 service for treble now

Change-Id: Ia45ae8d28070671f57bf146220fd067d46506341

sassmann committed:
lentis-common: cleanup scripts related to bluetooth

In style of
commit 61db41d598cb98c02cb29bf30536d99411ad9796
Author: Tony Layher <layhertony@gmail.com>
Date:   Tue Oct 21 23:40:44 2014 -0400

    trlte: common: update init files.  Kang klte for now.

Change-Id: I01e2fd2042da59e1b0a25c79571c3d3b1a144ded

github-cygwin committed:
lentis-common: net_bt_stack -> bluetooth

Signed-off-by: Corinna Vinschen <xda@vinschen.de>

github-cygwin committed:
lentis-common: fix wlan module path

Signed-off-by: Corinna Vinschen <xda@vinschen.de>

github-cygwin committed:
lentis-common: fix wpa_supplicant and macloader startup

Signed-off-by: Corinna Vinschen <xda@vinschen.de>

github-cygwin committed:
lentis-common: add wifi stuff to products

Signed-off-by: Corinna Vinschen <xda@vinschen.de>

github-cygwin committed:
lentis-common: start wcsnss_service, move to vendor

Signed-off-by: Corinna Vinschen <xda@vinschen.de>

sassmann committed:
lentis-common: fix permission for /sys/class/rfkill/rfkill0/device/extldo

Change-Id: I20952d5a6cec461045270363008cb83d4431fd46

sassmann committed:
lentis-common: re-add wcnss_filter

Change-Id: If34d52c132614b8d7b5ca4e8a490f6ed19bd8e0b

sassmann committed:
lentis-common: remove mmi-ril-daemon

ril is now started via /vendor/etc/init/rild.rc

Change-Id: Id0aec26916bc971b8f22e9458f9397778fa269ff

github-cygwin committed:
lentis-common: Move macloader to vendor

Signed-off-by: Corinna Vinschen <xda@vinschen.de>

github-cygwin committed:
lentis-common: create symlinks to bluetooth patch files

We moved the BT patchfiles to /system/vendor but ROME code has
/system/etc/firmware hardcoded

Signed-off-by: Corinna Vinschen <xda@vinschen.de>

github-cygwin committed:
lentis-common: build qcacld driver as builtin

Signed-off-by: Corinna Vinschen <xda@vinschen.de>

github-cygwin committed:
lentis-common: chown fwpath to allow hal file access

Signed-off-by: Corinna Vinschen <xda@vinschen.de>

github-cygwin committed:
lentis-common: drop macloader

kernel now reads MAC from /efs/wifi/.mac.info directly.

Signed-off-by: Corinna Vinschen <xda@vinschen.de>

github-cygwin committed:
lentis-common: default to LTE

Signed-off-by: Corinna Vinschen <xda@vinschen.de>

github-cygwin committed:
lentis-common: drop dpi setting here, it is device dependent

Signed-off-by: Corinna Vinschen <xda@vinschen.de>

github-cygwin committed:
lentis-common: create symlinks for libs and binaries, fix symlink creation

* /system/lib/libprotobuf-cpp-full.so is the OSS variant which is
  missing symbols.  Override it with a symlink to the vendor version.
  The symlink is not strictly necessary, but we must get rid of the
  OSS lib to avoid that it gets loaded first.

* mdm_helper and mdm_helper_proxy try to start /system/bin/ks, which
  has been moved to /system/vendor/bin.

* Make sure symlinks are created in wifi subdir.

Signed-off-by: Corinna Vinschen <xda@vinschen.de>

github-cygwin committed:
lentis-common: require system_prop.mk

Signed-off-by: Corinna Vinschen <xda@vinschen.de>

github-cygwin committed:
add ssh link so sshd finds empty dir

Signed-off-by: Corinna Vinschen <xda@vinschen.de>

sassmann committed:
lentis-common: add libaptX from Moto griffin - NCLS25.86-11-4-6-5

Change-Id: I68b7713535900df1441c3ca20186d0231c69eedc

micky387 committed:
lentis-common: remove audiod ,deprecated now

Signed-off-by: Corinna Vinschen <xda@vinschen.de>

micky387 committed:
lentis-common: Fix permission for camera

Signed-off-by: Corinna Vinschen <xda@vinschen.de>

micky387 committed:
lentis-common: Add STLport library as prebuilt binary

* libstlport.so has been deprecated from AOSP 8.0.0,
    and the sources have been replaced by prebuilt
    binaries only for selected official devices
* libstlport is a dependency of legacy camera
    or display HAL proprietary stacks
** Import libstlport.so from last cm-14.1 (lineage 7.1) build

Signed-off-by: Corinna Vinschen <xda@vinschen.de>

github-cygwin committed:
lentis-common: add symlinks to camera vendor files

Signed-off-by: Corinna Vinschen <xda@vinschen.de>

github-cygwin committed:
lentis-common: remove fingerprintd from PRODUCT_PACKAGES

Signed-off-by: Corinna Vinschen <xda@vinschen.de>

github-cygwin committed:
lents-common: Move fingerprint to apq8084-common

Signed-off-by: Corinna Vinschen <xda@vinschen.de>

github-cygwin committed:
lentis-common: fix sensor location

Signed-off-by: Corinna Vinschen <xda@vinschen.de>

github-cygwin committed:
lentis-common: Use nfc settings for new NFC impl in apq8084-common

Signed-off-by: Corinna Vinschen <xda@vinschen.de>

mikeNG committed:
lentis-common: Move permissions to vendor

Signed-off-by: Corinna Vinschen <xda@vinschen.de>

github-cygwin committed:
lentis-common: Move WLAN firmware to vendor

Signed-off-by: Corinna Vinschen <xda@vinschen.de>

github-cygwin committed:
lentis-common: Move sec_config to apq8084-common (src) and vendor (inst)

Signed-off-by: Corinna Vinschen <xda@vinschen.de>

sassmann committed:
lentis-common: clean-up rc files

Note that the hostapd service is defined in
/vendor/etc/init/hostapd.android.rc. Therefore removing it from
init.qcom.rc

Change-Id: Ie089b055e1ca85d610c6fa622ab93f91394f5ba4

sassmann committed:
lentis-common: remove unused init.sec.boot.sh

Change-Id: Iecde8fd10d4d9f59fc81e8a7a11428091b1363bd

sassmann committed:
lentis-common: remove unused init.qcom.bt.sh

Change-Id: I32667d0d505ee4b2b8e9b949e7d111653721e5aa

micky387 commented:
Do you have any benefict to change permissions of : 
/dev/diag                 0660   system     qcom_diag --> /dev/diag                 0660   radio      radio     ?

or simply change  all qcom_diag to oem_2950 now

irsc_util is used for sec_config

sassmann commented:
qcom_diag group no longer exists, so I'm removing it. Why change it to oem_2950?
The sec_config file does not exist for lentis and therefore irsc_util didn't start. Do you have one for trlte? Can you check if you need irsc_util?

github-cygwin commented:
dunno about oem_2950 but sec_config has been moved to apq8084-common yesterday
and gets installed into vendor.

sassmann committed:
lentis-common: no need to override ro.nfc.port

Property is set in init.qcom.rc when appropriate.

Change-Id: Ice32beafeb3f0c18815e848a36715d1ba59b60ff

sassmann committed:
lentis-common: start sshd on eng builds

ro.build.type is currently broken and returns "user" instead of "eng" on
eng builds. Using ro.build.flavor=lineage_lentislte-eng until
ro.build.type gets fixed.
Boosting priority as otherwise ssh is barely responsive if display is
off.

Change-Id: I3eb69f56df220a33d5a89174856737df7d3525a9

sassmann committed:
lentis-common: fix scaling_governor

No wonder that the system feels sluggish...

Change-Id: I2bce81b36af5cb22ab14ab21a56382db72169e6d

github-cygwin committed:
lentis-common: grab trlte-common Snap overlay

Signed-off-by: Corinna Vinschen <xda@vinschen.de>

sassmann committed:
lentis-common: move swap enable to boot_completed

Also decrease logging.

Change-Id: I96d51e6da63e6d8da8db61d24d8d61c07e075aaf
